src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -387,73 +438,210 @@ public void testTestPermissions() {
+    assertEquals(UPDATE_TIME, policyInfo.getUpdateTime());
+    assertEquals(VERSION, policyInfo.getVersion());
+  }
+
+  @Test
+  public void testGetEffectiveOrgPolicyWithException() {
+    String exceptionMessage = "Not Found";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
+        .when(resourceManagerMock)
+        .getEffectiveOrgPolicy(RESOURCE, CONSTRAINTS);
+    try {
+      resourceManagerMock.getEffectiveOrgPolicy(RESOURCE, CONSTRAINTS);
+      fail();
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testGetOrgPolicy() {
+    Mockito.when(resourceManagerMock.getOrgPolicy(RESOURCE, CONSTRAINTS))
+        .thenReturn(ORG_POLICY_INFO);
+    OrgPolicyInfo policyInfo = resourceManagerMock.getOrgPolicy(RESOURCE, CONSTRAINTS);
+    assertEquals(CONSTRAINTS, policyInfo.getConstraint());
+    assertEquals(BOOLEAN_POLICY, policyInfo.getBoolPolicy());
+    assertEquals(LIST_POLICY, policyInfo.getPolicies());
+    assertEquals(UPDATE_TIME, policyInfo.getUpdateTime());
+    assertEquals(VERSION, policyInfo.getVersion());
+  }
+
+  @Test
+  public void testGetOrgPolicyWithException() {
+    String exceptionMessage = "Not Found";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
+        .when(resourceManagerMock)
+        .getOrgPolicy(RESOURCE, CONSTRAINTS);
+    try {
+      resourceManagerMock.getOrgPolicy(RESOURCE, CONSTRAINTS);
+      fail();
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testListAvailableOrgPolicyConstraints() throws IOException {
+    ListResult<Constraint> expectedResult =
+        ListResult.of(CURSOR, ImmutableList.of(CONSTRAINT_INFO.toProtobuf()));
+    Mockito.when(resourceManagerRpcMock.listAvailableOrgPolicyConstraints(NAME, EMPTY_RPC_OPTIONS))
+        .thenReturn(expectedResult);
+    ListResult<Constraint> page =
+        resourceManagerRpcMock.listAvailableOrgPolicyConstraints(NAME, EMPTY_RPC_OPTIONS);
+    assertEquals(CURSOR, page.pageToken());
+    for (Constraint constraint : page.results()) {
+      assertEquals(NAME, constraint.getName());
+      assertEquals(CONSTRAINT_DEFAULT, constraint.getConstraintDefault());
+      assertEquals(DISPLAY_NAME, constraint.getDisplayName());
+      assertEquals(DESCRIPTION, constraint.getDescription());
+      assertEquals(
+          LIST_CONSTRAINT.getSuggestedValue(), constraint.getListConstraint().getSuggestedValue());
+      assertEquals(
+          LIST_CONSTRAINT.getSupportsUnder(), constraint.getListConstraint().getSupportsUnder());
+      assertEquals(VERSION, constraint.getVersion());
+    }
+  }
+
+  @Test
+  public void listAvailableOrgPolicyConstraintsWithException() {
+    String exceptionMessage = "Not Found";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
```

Comment:
You've mocked out the method you're trying to test.

Reply:
done